### ISSUE TITLE
Enable proper titles for Nodes

### ISSUE THREAD

**Issue by bep** [spf13/hugo#1051]
See

http://discuss.gohugo.io/t/how-to-name-list-templates/949/15
http://discuss.gohugo.io/t/key-in-site-taxonomies-tagname-urlized-by-default/1025

**Comment by SystemZ** [comment]
Is there any simple method to get normal, non lowercase string from $key variable as used in documentation? This small bug is driving me crazy for few hours now :disappointed: Variables in hugo are little confusing and I can't get $key by other method.

http://gohugo.io/taxonomies/displaying/
```
<section id="menu">
    <ul>
        {{ range $key, $taxonomy := .Site.Taxonomies.categories }}
        <li> {{ $key }} </li>
        <ul>
            {{ range $taxonomy.Pages }}
            <li hugo-nav="{{ .RelPermalink}}"><a href="{{ .Permalink}}"> {{ .LinkTitle }} </a> </li>
            {{ end }}
        </ul>
        {{ end }}
    </ul>
</section>
```

**Comment by bep** [comment]
@SystemZ see https://github.com/spf13/hugo/issues/1180 for some related commits -- which I believe is still in the DEV version of Hugo.

**Comment by bep** [comment]
Will be fixed by #2297.
